src/js/component/TodoListContainer/TodoContainer/TodoFilters.js

```diff
@@ -0,0 +1,50 @@
+export default function TodoFilters(parent, {onShowAll, onActive, onCompleted}) {
+  this.$parent = parent
+  this.dom;
+
+  this.setDom = () => {
+    this.dom = document.createElement('ul');
+    this.dom.className = 'filters';
+    this.$parent.append(this.dom);
+  }
+
+  this.setState = () => {
+    this.setDom();
+    this.dom.insertAdjacentHTML('afterbegin', this.template);
+    this.setEvent();
+  }
+
+  this.template = `
+  <li>
+    <a href="#all" class="selected">전체보기</a>
+  </li>
+  <li>
+    <a href="#active" class="active">해야할 일</a>
+  </li>
+  <li>
+    <a href="#completed" class="complete">완료한 일</a>
+  </li>
+  `
+
+  this.setEvent = () => {
+    const $all = document.querySelector('.selected');
+    const $active = document.querySelector('.active');
+    const $completed = document.querySelector('.complete');
+    $all.addEventListener('click',(event) => this.onShowAllHandler(event))
+    $active.addEventListener('click',(event) => this.onShowActiveHandler(event))
+    $completed.addEventListener('click',(event) => this.onShowCompletedHandler(event))
+
+  }
+
+  this.onShowAllHandler = () => {
+    onShowAll();
+  }
+
+  this.onShowActiveHandler = () => {
+    onActive();
+  }
+
+  this.onShowCompletedHandler = () => {
+    onCompleted();
+  }
```

Comment:
메서드에게 event 객체를 매개변수로 넘겨주지만 사용하지 않는 것 같습니다!
요렇게는 어떨까요..!
```suggestion
    $all.addEventListener('click',() => onShowAll())
    $active.addEventListener('click',() => onActive())
    $completed.addEventListener('click',() => onCompleted())
  }

```

Reply:
훨씬더 간단하네요!! 바로 수정해야겠습니다 감사합니다!!